### PR TITLE
Augk mar25 editproducts

### PR DESCRIPTION
**Pull request by augustine24**
Today I added just a bit of detail, I haven't gotten the add to cart button going, but I added a order quantity button that I will change later, and added a name variable. I also added a little bit of text at the top of the cart page so users know they're on the cart page like they do in the products page. 

Edit: Here's how it looks with the changes

Screenshot 2024-03-26 142704

### CODE REVIEW

File: frontend/package.json
Comment (sarmadH97) [inline frontend/package.json:1]: Already present in the file, please remove this duplication.

**Comment by sarmadH97** [comment]
@augustine24  Please attach the screenshot of the UI changes to the PR description as well. 
Why are there two files products.js with similar code? Please take a look and sort the files out accordingly. 
Thanks.

**Review by sarmadH97 — COMMENTED** [review]
Resolve the added comments.

File: frontend/src/pages/products/products.js
Comment (sarmadH97) [inline frontend/src/pages/products/products.js:1]: Duplicate file.

Comment (augustine24) [reply frontend/src/pages/products/products.js:1]: I've fixed the issue and I'll attach a picture of the front end with the PR request

File: frontend/package.json
Comment (augustine24) [reply frontend/package.json:1]: Removed the duplication I'll make the PR request again with the changes

**Review by sarmadH97 — APPROVED** [review]

**Pull request merged**